app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -666,6 +668,7 @@ impl Area {
                     // === Press ===
 
                     eval_ mouse_down ([crumbs,frp] frp.source.on_port_press.emit(&crumbs));
+                    eval_ frp.test_port_press ([crumbs,frp] frp.source.on_port_press.emit(&crumbs));
```

Comment:
Repeated logic, don't do it in FRP. Instead, merge both event streams into one, and then eval the logic.